cpu/esp8266/ld/esp8266.riot-os.ld

```diff
@@ -242,7 +242,7 @@ SECTIONS
     *freertos/*(.literal .text .literal.* .text.*)
     *freertos_common/*(.literal .text .literal.* .text.*)
     *periph/*(.literal .text .literal.* .text.*)
-    *xtimer/*(.literal .text .literal.* .text.*)
+    *ztimer/*(.literal .text .literal.* .text.*)
```

Comment:
This would have to be
```diff
-     *ztimer/*(.literal .text .literal.* .text.*)
+    *ztimer_core/*(.literal .text .literal.* .text.*)
```
I didn't know that objects are not stored in `ztimer` but in `ztimer_core` although the module is called `ztimer`. But this alone doesn't solve the problem, unfortunatly :worried: